internal/provider/data_source_packer_image_test.go

```diff
@@ -134,3 +170,78 @@ func TestAcc_dataSourcePackerImage_revokedIteration(t *testing.T) {
 		},
 	})
 }
+
+func TestAcc_dataSourcePackerImage_channelAndIterationIDReject(t *testing.T) {
+	fingerprint := "rejectIterationAndChannel"
+	configs := []string{
+		testAccPackerImageBothChanAndIter,
+		testAccPackerImageBothChanAndIterRef,
+	}
+
+	for _, cfg := range configs {
+		resource.Test(t, resource.TestCase{
+			PreCheck:          func() { testAccPreCheck(t, map[string]bool{"aws": false, "azure": false}) },
+			ProviderFactories: providerFactories,
+			Steps: []resource.TestStep{
+				// basically just testing that we don't pass validation here
+				{
+					PlanOnly: true,
+					PreConfig: func() {
+						deleteChannel(t, acctestArchImageBucket, acctestImageChannel, false)
+						deleteIteration(t, acctestArchImageBucket, fingerprint, false)
+						deleteBucket(t, acctestArchImageBucket, false)
+
+						upsertRegistry(t)
+						upsertBucket(t, acctestArchImageBucket)
+						upsertIteration(t, acctestArchImageBucket, fingerprint)
+						itID, err := getIterationIDFromFingerPrint(t, acctestArchImageBucket, fingerprint)
+						if err != nil {
+							t.Fatal(err.Error())
+						}
+						upsertBuild(t, acctestArchImageBucket, fingerprint, itID)
+						createChannel(t, acctestArchImageBucket, acctestImageChannel, itID)
```

Comment:
Can this be done outside `resource.Test` before the loop? These can be done just once and it will work for both test cases.

Reply:
Oh, I think I had a similar problem before! No problem then. Thanks for having a look.

Reply:
Oh, I think I had a similar problem before! No problem then. Thanks for having a look.